example/data/collections/s3_MOD09GQ_006-unique/s3_MOD09GQ_006.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
This is a copy of the existing 9GQ collection, modified  such that the generated cmr_xml file from the fake processing step works with downstream tasks. 

This is good scaffolding but should be modified/changed depending on what happens in CUMULUS-4085

Reply:
Except it doesn't, in this case  because our fake-processing script (that generates the .cmr.xml)  uses the incoming granule_id instead of the producerGranuleId to generate the filename. 

There's work here to do this, but it probably feels very much either scoped to the the integration test effort, as that effort should be adding post-workflow assertions/etc.